source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/date-range/index.tsx

```diff
@@ -33,44 +37,58 @@ class DateRange extends React.Component<IOwnProps & i18nProps> {
     return day < compare || day > maxDate;
   }
 
+  handleWith = (handler) => (...eh) => {
```

Comment:
if is not going to be used add a todo explaining why or remove it

Reply:
oops